etc/settings.yaml.in

```diff
@@ -7,6 +7,10 @@ locations:
   krona-db-dir: databases/krona
   sigmut-db-dir: databases/sigmut
   vep-db-dir: databases/vep
+#for trimming the raw reads a min_len argument is set. Only reads longer than min_len are selected. By default min_len is 120 bp (read-length * cut_off)
```

Comment:
I would prefer not to mention "min_len" --- that's an implementation detail.  Rather spell it all out.

Here's a draft:

These settings are used to filter raw reads when trimming.  Reads that are shorter than the product of `read-length` (the length in basepairs) and the `cut-off` factor are removed.